write-api-client/README.md

```diff
@@ -17,6 +17,9 @@ repo=test-repository
 write-api-url=http://localhost:8082/api/ga4gh/v1
 ```
 
+After building, the CLI's jar file can be found under at the directory:  write-api/write-api-client/target/
+
```

Comment:
Let's add some scenarios. 
Add a few example CLI invocations and explain the result.